google_cloud_storage/index.js

```diff
@@ -23,5 +28,42 @@ async function uploadFile(input) {
     await bucket.upload(filePath, {
 
     });
+
+    if (deleteOnUpload) {
+        fs.unlink(filePath, (err) => {
+            if (err) {
+                console.error("file was uploaded but did not get deleted from the local filesystem");
+            }
+            console.info("file deleted");
+        });
+    }
     return {"Success" : true };
+}
+
+async function downloadFile(input) {
+    apptree.validateInputs('Credential', 'FileName', 'ProjectId', 'Bucket', 'ServerFilePath');
+    let credential = input['Credential'];
+    let fileName = input['FileName'];
+    const bucketName = input['Bucket'];
+    const serverDir = input['ServerFilePath'];
```

Comment:
Change it to just `FilePath`. Remove `FileName`

ie.

FilePath: "/users/Matthew/desktop/test.txt"

Reply:
updated